tasks/master.yml

```diff
@@ -1,15 +1,13 @@
 ---
+
 - name: Install the master server
-  shell: ipa-server-install -r {{ ansible_domain | upper }} -n {{ ansible_domain }} -p {{ directory_admin_password }}  -U -a {{ kerberos_admin_password }} --ip-address {{ ansible_default_ipv4.address }}
+  shell: >
+    ipa-server-install -U \
+      -r {{ freeipa_domain | upper }} -n {{ freeipa_domain }} \
+      -p {{ directory_admin_password | quote }} -a {{ kerberos_admin_password | quote }} \
+      --hostname {{ inventory_hostname }} --ip-address {{ frontnet_addr }} \
+      -N --setup-dns --auto-forwarders --forward-policy only
```

Comment:
For the sake of readability I'd put each arg on a new line.  You can use 'command' or 'shell' without the backslash line extensions too as the '>' will just concat the lines together with a space between them.  See: https://github.com/nexcess/ansible-playbook-influxdb-enterprise/blob/master/playbooks/tasks/data.yml#L127